option optimize_for = LITE_RUNTIME;

message KingFightStartRequest
{
 required uint32 id = 1;
 required uint32 star = 2;
}
